Source/Core/DolphinWX/Cheats/CheatSearchTab.cpp

```diff
@@ -42,7 +42,7 @@ CheatSearchTab::CheatSearchTab(wxWindow* const parent)
 	m_btn_next_scan->Disable();
 
 	// data sizes radiobox
-	std::array<wxString, 3> data_size_names = {{ _("8-bit"), _("16-bit"), _("32-bit") }};
+	std::array<wxString, 3> data_size_names = { { _("8-bit"), _("16-bit"), _("32-bit") } };
```

Comment:
This PR is meant to consist almost entirely of whitespace changes and formatting, rather than actual code changes.